proof/refine/ARM/KHeap_R.thy

```diff
@@ -4188,4 +4188,42 @@ lemma pred_tcb_at'_equiv:
       clarsimp simp: pred_tcb_at'_def pred_map_def obj_at'_real_def ko_wp_at'_def projectKOs
                      opt_map_def)
 
+lemma isBlockedOnSend_equiv:
+  "isBlockedOnSend st = is_BlockedOnSend st"
```

Comment:
Definitely not necessary as part of this PR, but it might be worth making these sort of rules `[simp]` as a first step towards consolidating these definitions.